changelogs/fragments/batch_mode_elasticsearch_plugin.yaml

```diff
@@ -0,0 +1,4 @@
+---
+minor_changes:
+  - Add the possibility to use the elasticsearch_plugin installation batch mode
```

Comment:
Start this entry with `- elasticsearch_plugin - `. This make the module name easier to find when visually scanning the changelogs.